react/Minicart.tsx

```diff
@@ -20,6 +20,7 @@ interface MinicartProps {
   maxDrawerWidth: number | string
   drawerSlideDirection: SlideDirectionType
   quantityDisplay: MinicartIconButtonType
+  showTotalItemsQty: boolean
```

Comment:
We tend to stay away from new `boolean` props, since they limit us in the implementation of future features and increases the API surface area very quickly.

A more extensible solution to this might be a `quantityCount` prop, which is an `enum`, with values such as `"total" | "unique"` or `"total" | "distinct"`

Reply:
Make sense, even if I do not see any other option...